app/api/resources/rapid_view.py

```diff
@@ -0,0 +1,80 @@
+from urllib.parse import parse_qs
+from fastapi import APIRouter, Request, HTTPException, Query
+from fastapi.responses import RedirectResponse
+import logging
+import re
+from core.logging import InterceptHandler
+from core.config import ENSEMBL_URL
+from api.utils.metadata import get_genome_id_from_assembly_accession_id
+from api.utils.rapid import format_assembly_accession
+
+logging.getLogger().handlers = [InterceptHandler()]
+
+router = APIRouter()
+
+
+# Resolve rapid urls
+@router.get("/{species_url_name}", name="Rapid Species Resources")
+@router.get("/{species_url_name}/", name="Rapid Species Resources")
```

Comment:
What's the difference between line 17 and 18? is it just because of / ?

Reply:
yes it is because of the trailing slash /